Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -4176,8 +4176,10 @@ void WebPageProxy::accessibilitySettingsDidChange()
     if (!hasRunningProcess())
         return;
 
+#if PLATFORM(COCOA)
```

Comment:
I think it would be reasonable to just do this (even all the IPC) on all platforms. It is unlikely to be a useful optimization and just makes things harder to read.